packages/shared/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "version": "0.0.0-dev+semantically-released",
+  "description": "",
+  "keywords": [
+    "react",
+    "components"
+  ],
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index",
+  "homepage": "https://github.com/dcos-labs/ui-kit#readme",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dcos-labs/ui-kit.git"
+  },
+  "dependencies": {
+    "@dcos/ui-kit-design-tokens": "^0.0.0-dev+semantically-released",
+    "@dcos/ui-kit-utilities": "^0.0.0-dev+semantically-released"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1",
```

Comment:
Why only allow test runs from the root? Couldn't they all just run with jest?

Reply:
Would you rather run tests once or for each package? If we run it from root it is executing all tests, with the right scope